devtools/db_import/db_import/config.py

```diff
@@ -4,4 +4,55 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+import pathlib
+import yaml
+import contextlib
+import os
```

Comment:
nit: Sort in alphabetical order
```suggestion
import contextlib
import os
import pathlib
import yaml
```